parser-typechecker/src/Unison/Runtime/MCode.hs

```diff
@@ -894,13 +894,13 @@ emitLet
   -> Emit Section
 emitLet _   _   _   _ _   _   (TLit l)
   = fmap (Ins $ emitLit l)
-emitLet rns grp _   _ _   ctx (TApp (FComb r) args)
-  -- We should be able to tell if we are making a saturated call
-  -- or not here. We aren't carrying the information here yet, though.
-  | False -- not saturated
```

Comment:
We can no longer use a `| False` pattern guard to fake out the dead code detector.

Reply:
`not True` probably works :)

Reply:
Hehe I thought about doing that; I commented it out instead.